packages/story-editor/src/components/canvas/useInsertTextSet.js

```diff
@@ -94,10 +94,10 @@ function useInsertTextSet(shouldUseSmartColor = false) {
         const allColorsEqual = textElementsContrasts.every(
           (contrast) =>
             contrast === null ||
```

Comment:
If getting the image data fails from the generated image, it returns `{ color: null }` which was not considered here and failed when trying to get `r`, `g`, `b`, `a` from `null` in case it happened.